README.md

```diff
@@ -167,6 +169,8 @@ Use the below schema to configure Splunk Connect for Kafka
 | `splunk.validation.disable` | Disable validating splunk configurations before creating task. | `false` |
 | `splunk.hec.ssl.validate.certs` | Valid settings are `true` or `false`. Enables or disables HTTPS certification validation. |`true`|
 | `splunk.hec.http.keepalive` | Valid settings are `true` or `false`. Enables or disables HTTP connection keep-alive. |`true`|
+| `splunk.hec.http.proxy.host` | Http proxy host name. | `""` |
```

Comment:
Maybe we could add a short description here, similar to this from the issue description, which will explain when and why this proxy can be used?